Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Entries/RootAssetBrowserEntry.h

```diff
@@ -89,6 +90,7 @@ namespace AzToolsFramework
                 AZStd::unordered_set<AssetBrowserEntry*>& visitedSet);
 
             bool m_isInitialUpdate = false;
+            bool inProjectDirectory = false;
```

Comment:
```suggestion
            bool m_inProjectDirectory = false;
```

Reply:
I'm using the boolean so that `m_fullPath` isn't reset if we see continuous entries in a row that are relative to the project/engine directory if the directory has already been set correctly

However, it would probably make more sense to just check whether `parent->m_fullPath` is already relative to the correct directory